### ISSUE TITLE
Add-on does not automatically restart on startup 

### ISSUE THREAD

**Issue by maverik80** [Bilan/connectlife-api-connector#5]
I'm testing this add-on and it's really compliments, it was what I was looking for, but I had to restart my vm in proxmox for other reasons and the add-on didn't restart automatically

**Comment by maverik80** [comment]
now yes but I don't understand why it stops honestly

**Comment by maverik80** [comment]
So with watchdog it works fine and never stops, while if I turn off watchdog I get the problem, checked my broker but I have no errors, in your post on the forum I read about the api token? But in Redme you don't mention it, in case it is needed, where should the token recovered from the swaggen UI be put?

**Comment by alexcroce** [comment]
i hava the same issue. I turned on watchdog, but when i restart my haos, i see the same error in log file and i need to start manually,

**Comment by alexcroce** [comment]
@Bilan , i already updated. i will try to restart haos. Thank you for your work!

**Comment by alexcroce** [comment]
ok, now the add-on start, but i can't see my hisense splits . It says mqtt doesn't recognize them.

**Comment by alexcroce** [comment]
yes, splits are online, after restart of mosquito and connectlife app they are online again

**Comment by timewolf** [comment]
i ve a similar issue.
ive docker on an openmediavault system, configured docker for connectilfe, if I reboot omv the only docker that will fail is connectlife, if I do a down/up it works.
